README.md

```diff
@@ -4,19 +4,19 @@ This app runs the front-end published sites for all tiny news organizations, usi
 
 ## What's in here?
 
-- __\_\_mocks\_\___: Basic boilerplate for tests
-- __\_\_tests\_\___: Hasura API unit tests
-- __cached__: A folder for keeping cached API responses. Kept intentionally empty in version control.
-- __components__: All of the reusable React components used across the next.js frontend
-- __content__: Markdown-based content used on the frontend.
-- __cypress__: Test infrastructure for Cypress, which runs integration tests.
-- __data__: Obsolete.
-- __hasura__: A git submodule controlling our Hasura instances.
-- __lib__: Code that interacts with our GraphQL API and other third-party APIs, such as Letterhead.
-- __pages__: Page-level components used to build the site structure in next.js
-- __public__: Various static files that need to get served publicly.
-- __script__: Command-line scripts used for various tasks, like bootstrapping new organizations.
-- __styles__: Extra styles that don't conform well to using Tailwind.
+- **\_\_mocks\_\_**: Basic boilerplate for tests
```

Comment:
This was auto-formatted, I didn't make this change manually. It seems to be an [alternative](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text) to `__` for bold text.